app/src/main/res/values/bool.xml

```diff
@@ -0,0 +1,4 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+    <bool name="dimelo_toolbar_display_title">true</bool>
```

Comment:
default value of `dimelo_toolbar_display_title` is `true` so you can remove this file or change the value to `false`

Reply:
ping @dhouha123